check_and_format.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+pushd headers
+make
+make format-check
+make symbol-check
+#make symbol-check-extra
+popd
+
+cargo run -- check -r -Vvbomu -d screaming_snake_case -f pascalcase symbols/**.yml
```

Comment:
I just realized I forgot to update the docs, but this should be:

```suggestion
cargo run --release -- check -r -Vvbomu -d screaming_snake_case -f pascalcase -f snake_case symbols/*.yml
```